pydoc/quickstart.rst

```diff
@@ -4,10 +4,51 @@
 Quickstart
 #######################
 
-This guide assumes you have satisfied the `Requirements <setup/REQUIREMENTS.html>`_ and followed the :ref:`installation-label` to build the arkouda server. Also, both your ``PATH`` and ``PYTHONPATH`` environment variables should contain the arkouda root directory.
+This guide is intended to instruct users on the installation Arkouda. It will also walk through launching and shutting down the server. For usage information, visit our :ref:`usage-label`
 
 **********************
-Launch Arkouda Server
+Install Dependencies
+**********************
+
+1. Follow the `Chapel Quickstart Guide <https://chapel-lang.org/docs/usingchapel/QUICKSTART.html>`_
+2. Follow the `Anaconda Installtion Guide <https://docs.anaconda.com/anaconda/install/index.html>`_
+3. Configure your conda environment
+
+   .. substitution-code-block:: bash
+
+       conda env create -f arkouda-env.yml
+
+**********************
+Install Arkouda
+**********************
+
+1. Download Arkouda `v2023.01.11 <https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2023.01.11.tar.gz>`_
+2. Unpack the source files:
+
+   .. substitution-code-block:: bash
+
+       tar xzf arkouda-2023.01.11.tar.gz
+
+3. Change to the arkouda directory
+
+   .. substitution-code-block:: bash
+
+       cd arkouda-2023.01.11
```

Comment:
is this something we should try to update each release?

Reply:
@pierce314159 - I mentioned in the description that after each release, we will need to update the download link and page to match the latest release. 

Maybe we should add the release process to the developer section and include doing this in the workflow?